src/plugins/shell/shell_impl.cpp

```diff
@@ -73,7 +73,9 @@ bool ShellImpl::send_command_message(std::string command)
             timeout_ms,
             0,
             static_cast<uint8_t>(MAVLINK_MSG_SERIAL_CONTROL_FIELD_DATA_LEN),
-            reinterpret_cast<const uint8_t*>(command.c_str()));
+            reinterpret_cast<const uint8_t*>(command.c_str()),
+            _parent->get_system_id(),
+            _parent->component_ids().back());
```

Comment:
I'm sure this is wrong, but I didn't have a better idea for what might be right.

Reply:
It should be `_parent->get_autopilot_id()`.

Reply:
thanks, change has been made